public/app.js

```diff
@@ -1,6 +1,36 @@
-html {
-  box-sizing: border-box;
-}
-*, *:before, *:after {
-  box-sizing: inherit;
-}
+(function() {
+  var time = document.querySelector('.time');
+  var line = document.querySelector('.line');
+  var colorContainer = document.querySelector('.container');
+  var colorHex = document.querySelector('.color');
+
+  window.setInterval(() => {
+    var d = new Date();
+    console.log(d);
+    var nowHour = padZeros(d.getHours());
+    var nowMin =  padZeros(d.getMinutes());
+    var nowSec =  padZeros(d.getSeconds());
+    time.innerHTML = `${nowHour}:${nowMin}:${nowSec}`;
+
+    var colorHour = toColorRange(nowHour, 60);
+    var colorMin = toColorRange(nowMin, 60);
+    var colorSec = toColorRange(nowSec, 60);
+    var colorApply = `rgb(${colorHour}, ${colorMin}, ${colorSec})`;
+    colorContainer.style.background = colorApply;
+    colorHex.innerHTML = colorApply;
+    line.style.width = (nowSec / 60 * 40 + 'rem');
```

Comment:
Interesting math. I like the creativity of using this to stop the line bar's max width. Ideally that would be done with CSS though on a parent element.